contracts/StablecoinConverter.sol

```diff
@@ -249,18 +255,17 @@ contract StablecoinConverter is EpochTokenLocker {
         }
     }
 
-    function getExecutedBuyAmount(
-        uint128 executedSellAmount,
+    function getExecutedSellAmount(
+        uint128 executedBuyAmount,
         uint128 buyTokenPrice,
         uint128 sellTokenPrice
     ) internal view returns (uint128) {
-        uint128 buyAmount = uint128(executedSellAmount.mul(sellTokenPrice) / buyTokenPrice);
-        // executedBuyAmount = buyAmount * (1 - (1/feeDenominator))
-        //                   = buyAmount - buyAmount/feeDenominator (*)
-        //                   = (buyAmount * feeDenominator)/ feeDenominator - buyAmount/feeDenominator
-        //                   = (buyAmount * feeDenominator - buyAmount) / feeDenominator
-        //                   = (buyAmount * (feeDenominator - 1)/feeDenominator
-        return uint128(buyAmount.mul(feeDenominator - 1) / feeDenominator);
+        // executedSellAmount = sellAmount * (1 - (1/feeDenominator))
```

Comment:
This comment explains how the fee is calculated. However it doesn't explain the weird order of division. Maybe we should comment on that too?